src/foundry/client/pixi/layers/effects/visibility.d.mts

```diff
@@ -33,7 +32,10 @@ declare global {
      * }
      * ```
      */
-    visionModeData: { source: foundry.canvas.sources.PointVisionSource.Any | null; activeLightingOptions: Record<string, unknown> };
+    visionModeData: {
+      source: foundry.canvas.sources.PointVisionSource.Any | null;
+      activeLightingOptions: Record<string, unknown>;
```

Comment:
Please use `AnyObject` instead. Also can we be more concrete about the type here?

Reply:
Looks like it's used to diff against the vision mode
```js
    const lightingOptions = this.visionModeData.source?.visionMode.lighting || {};
    const diffOpt = foundry.utils.diffObject(this.visionModeData.activeLightingOptions, lightingOptions);
    ```

Reply:
Do you think they should have the same shape as `lightingOptions` then?

Reply:
Yeah, so in the last commit I swapped to `VisionMode["_source"]["lighting"]`